tests/e2e/config/bootstrap.js

```diff
@@ -200,7 +200,7 @@ beforeAll( async () => {
 	observeConsoleLogging();
 	await setBrowserViewport( 'large' );
 	await page.setDefaultNavigationTimeout( 10000 );
-	await page.setDefaultTimeout( 3000 );
+	await page.setDefaultTimeout( 10000 );
```

Comment:
I realized the cause for most of the recent flaky E2E tests were related to async actions not completing in time (such as installing a theme). Raising the default timeout to 10s seems like a reasonable amount and has solved the flaky tests I sometimes encounter locally.